code/modules/projectiles/updated_projectiles/ammo_datums.dm

```diff
@@ -1480,20 +1552,20 @@
 	G.damtype = "fire"
 	G.SetLuminosity(G.brightness_on)
 
-/datum/ammo/rocket/nobugs
-	name = "\improper NO BUGS rocket"
+/datum/ammo/rocket/toy
```

Comment:
unrelated change, should be separate pr

Reply:
I agree, separating it with my shitty commits wouldn't be easy though. I'll watch for this sort of thing in the future.